### ISSUE TITLE
Soundcloud Not Working - 401 HTTP Error 401: Unauthorized

### ISSUE THREAD

**Issue by djace** [rg3/youtube-dl#6971]
Hi,

All SoundCloud URLs are failing with 401 Unauthorized:

```
youtube-dl --verbose http://soundcloud.com/m28xxlover/nervo-live-at-electric-zoo-2015-new-york-05-sep-2015                                                                                         
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'--verbose', u'http://soundcloud.com/m28xxlover/nervo-live-at-electric-zoo-2015-new-york-05-sep-2015']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.09.22
[debug] Python version 2.7.10 - Linux-3.16.0-4-amd64-x86_64-with-debian-stretch-sid
[debug] exe versions: ffmpeg N-45620-gcf16b45-, ffprobe N-45620-gcf16b45-
[debug] Proxy map: {}
[soundcloud] m28xxlover/nervo-live-at-electric-zoo-2015-new-york-05-sep-2015: Resolving id
[soundcloud] m28xxlover/nervo-live-at-electric-zoo-2015-new-york-05-sep-2015: Downloading info JSON
ERROR: Unable to download JSON metadata: HTTP Error 401: Unauthorized (caused by HTTPError()); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 326, in _request_webpage
    return self._downloader.urlopen(url_or_request)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1865, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "/usr/lib/python2.7/urllib2.py", line 437, in open
    response = meth(req, response)
  File "/usr/lib/python2.7/urllib2.py", line 550, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python2.7/urllib2.py", line 475, in error
    return self._call_chain(*args)
  File "/usr/lib/python2.7/urllib2.py", line 409, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 558, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
```
The problem seems to be that all API calls with that client id will fail (such as the first one, to get the JSON from http://api.soundcloud.com/resolve.json?url=http://soundcloud.com/m28xxlover/nervo-live-at-electric-zoo-2015-new-york-05-sep-2015&client_id=b45b1aa10f1ac2941910a7f0d10f8e28) because the `b45b1aa10f1ac2941910a7f0d10f8e28` client ID is exceeding the bandwidth or maybe it has simply been banned.

I've tried using client id `986b39d4513a5b501d57d973318715f0` (from https://github.com/soundcloud/soundcloud-javascript/issues/12) and the URL returns a valid response.

**Comment by yan12125** [comment]
Thanks for debugging. It's already fixed in eb11cbe8674705647d6bd2947a44e08543663633.

**Comment by djace** [comment]
Ouch, I deserve that for searching only the open issues and not bothering with the closed ones. Sorry!
